src/main/java/io/jenkins/plugins/orka/client/OrkaClient.java

```diff
@@ -168,7 +181,14 @@ private Builder getAuthenticatedBuilder(String url) throws IOException {
     private String executeCall(Request request) throws IOException {
         logger.fine("Executing request to Orka API: " + '/' + request.method() + ' ' + request.url());
 
-        try (Response response = client.newCall(request).execute()) {
+        OkHttpClient httpClient = client;
```

Comment:
I have an idea for improving this:

What do you think of initializing the instance specific client inside the ctor.
Because the OrkaClient has 1 timeout and it never changes, you can init the http client in the ctor and reuse it across all calls.
It is a bit redundant to do it prior to each call.

This time you will not have to keep the `httpClientTimeout` as a field as well.